enumerables.rb

```diff
@@ -0,0 +1,154 @@
+module Enumerable
+  def my_each
+    each do |value|
+      yield(value)
+    end
+  end
```

Comment:
* [ ] `my_each` is identical and returns the same thing as ruby's [`each`](https://ruby-doc.org/core-2.6.4/Array.html#method-i-each). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_each.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))

* [ ] my_each returns an Enumerator if no block is given
  ```ruby
    array = [1,2,5,8,7,98,4]
    p array.my_each.class # not Enumerator
  ```